drivers/usb/uhc/Kconfig.mcux

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config USB_UHC_NXP_EHCI
+	bool "NXP MCUX USB EHCI Host controller driver"
+	default y
+	depends on DT_HAS_NXP_UHC_EHCI_ENABLED
+	select USB_UHC_NXP_MCUX
+	help
+	  NXP MCUX USB Host Controller Driver for EHCI.
+
+config USB_UHC_NXP_KHCI
+	bool "NXP MCUX USB KHCI Host controller driver"
+	default y
+	depends on DT_HAS_NXP_UHC_KHCI_ENABLED
+	select USB_UHC_NXP_MCUX
+	help
+	  NXP MCUX USB Host Controller Driver for KHCI.
+
+config USB_UHC_NXP_MCUX
+	bool
+	# Hidden option to simplify MCUX uhc requirement
+	default n
```

Comment:
We should not need `default n` here, it should be the default if this Kconfig is not selected.